test/callbacks/test_callbacks.py

```diff
@@ -1,10 +1,20 @@
+from sympy import Dummy
```

Comment:
do you really need `sympy`? it's breaking the tests, because it's not installed, and while those are the tests and it's fine adding a dependency there you don't seem to be using it...